graph/tags/tags.go

```diff
@@ -6,8 +6,10 @@ import (
 	"github.com/docker/distribution/registry/api/v2"
 )
 
+// Define DEFAULTTAG
```

Comment:
what about something like `DEFAULTTAG defines "latest" as the tag for images with implicit tag`